project/plugins.sbt

```diff
@@ -1,2 +1 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
\ No newline at end of file
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
```

Comment:
Do we need to add the project folder isn the repo ? I think this is specific to IDE and should not be part of the source repo. Thoughts ?

Reply:
I think `project` folder is part of scala project hierarchy. We need to add `plugins.sbt` under this folder. `plugins.sbt` is used to add all sort of plugins. Like we add maven plugins in the pom and then there is also `assembly.xml`, similarly in scala project, we can use this `sbt-assembly` plugin to create the fat jar.